examples/cross-mingw/shell.nix

```
@@ -0,0 +1,17 @@
+# See docs/cross_compilation.md for details.
+(import <nixpkgs> {
+  crossSystem = {
+    config = "x86_64-w64-mingw32";
+  };
```

```suggestion
  crossSystem = "x86_64-w64-mingw32";
```
Simple config can be just written as a string.

---

Actually doing so seemed to break the splicing? If I do the suggested change, it tries to use `mkShell` for `x86_64-w64-mingw32`.

---

Don't you just want `x86_64-w64-mingw32`? I tested it and it also works fine.

---

Oh it actually doesn't work. My bad.